mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -36,17 +43,64 @@ CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManage
+		COMMENT Retrieves a registry from this manager,
+		COMMENT or throws an exception when the registry does not exist.
+		COMMENT
+		COMMENT @throws IllegalStateException if the registry does not exist
 		ARG 1 key
 	METHOD method_33310 getOptional (Lnet/minecraft/class_5321;)Ljava/util/Optional;
 		ARG 1 key
+	METHOD method_34849 (Lcom/mojang/serialization/DataResult$PartialResult;)V
+		ARG 0 partialResult
 	METHOD method_39674 getInfos ()Ljava/lang/Iterable;
-	METHOD method_40302 (Lnet/minecraft/class_2378;)Lnet/minecraft/class_5455$class_6890;
+	METHOD method_40299 createDynamicRegistryManager (Lcom/mojang/serialization/Dynamic;)Lnet/minecraft/class_5455;
+		ARG 0 dynamic
+	METHOD method_40300 (Lcom/mojang/serialization/Dynamic;Lnet/minecraft/class_5321;)Lnet/minecraft/class_2378;
+		ARG 1 registryRef
+	METHOD method_40301 createCodec (Lcom/mojang/serialization/codecs/UnboundedMapCodec;)Lcom/mojang/serialization/Codec;
+		ARG 0 originalCodec
+	METHOD method_40302 of (Lnet/minecraft/class_2378;)Lnet/minecraft/class_5455$class_6890;
 		ARG 0 registries
+	METHOD method_40303 (Lnet/minecraft/class_5455$class_5456;)Lcom/mojang/serialization/Codec;
+		ARG 0 info
+	METHOD method_40304 (Lnet/minecraft/class_5455$class_6892;)Z
+		ARG 0 entry
+	METHOD method_40305 (Lnet/minecraft/class_5455;)Ljava/util/Map;
+		ARG 0 dynamicRegistryManager
+	METHOD method_40306 (Lnet/minecraft/class_5321;Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
+		ARG 1 codec
+	METHOD method_40307 createRegistry (Lnet/minecraft/class_5321;Lcom/mojang/serialization/Dynamic;)Lnet/minecraft/class_2378;
+		ARG 0 registryRef
+		ARG 1 dynamic
+	METHOD method_40308 streamManagedRegistries ()Ljava/util/stream/Stream;
+	METHOD method_40309 (Lnet/minecraft/class_2378;)Lcom/mojang/serialization/DataResult;
+		ARG 0 registry
+	METHOD method_40310 (Lnet/minecraft/class_5455$class_6892;)Lnet/minecraft/class_2378;
+		ARG 0 entry
+	METHOD method_40311 streamAllRegistries ()Ljava/util/stream/Stream;
+	METHOD method_40312 (Lnet/minecraft/class_5455$class_6892;)Lnet/minecraft/class_5321;
+		ARG 0 entry
+	METHOD method_40313 streamSyncedRegistries ()Ljava/util/stream/Stream;
+	METHOD method_40314 createAndLoad ()Lnet/minecraft/class_5455$class_6893;
+	METHOD method_40315 shouldSkipLoading (Lnet/minecraft/class_5321;)Z
+		ARG 0 registryRef
+	METHOD method_40316 toNonMutable ()Lnet/minecraft/class_5455$class_6890;
+	METHOD method_40317 createSimpleRegistry (Lnet/minecraft/class_5321;)Lnet/minecraft/class_2385;
+		ARG 0 registryRef
+	METHOD method_40318 streamStaticRegistries ()Ljava/util/stream/Stream;
+	METHOD method_40319 getNetworkEntryCodec (Lnet/minecraft/class_5321;)Lcom/mojang/serialization/DataResult;
+		ARG 0 registryKey
+	METHOD method_40320 createCodec ()Lcom/mojang/serialization/Codec;
+	METHOD method_40321 streamSyncedManagedRegistries ()Ljava/util/stream/Stream;
+	METHOD method_40323 collectMutableRegistries ()Ljava/util/Map;
```

Comment:
```suggestion
	METHOD method_40323 createMutableRegistries ()Ljava/util/Map;
```
This doesn't collect existing registries. I think `collect` is an implementation detail for creating empty mutable registries.